frontend/RaffleBay/RaffleBay/View/UploadSaleItemView.swift

```diff
@@ -131,7 +131,7 @@ struct UploadSaleItemView : View {
                     }){
                         Text("Submit")
                         .blueButtonText()
-                        .frame(minWidth:0, maxWidth: frameMaxWidth)
+                            .frame(minWidth:CGFloat(0), maxWidth: frameMaxWidth)
```

Comment:
i think this change was unecessary?

Reply:
Yeah idk it was one of the swiftUI errors that was given to me when there was really something else wrong